Mathlib/Algebra/BigOperators/Expect.lean

```diff
@@ -391,14 +405,19 @@ See `Equiv.expect_comp` for a version without `h`. -/
 lemma expect_equiv (e : ι ≃ κ) (f : ι → M) (g : κ → M) (h : ∀ i, f i = g (e i)) :
     𝔼 i, f i = 𝔼 i, g i := expect_bijective _ e.bijective f g h
 
-lemma expect_const [Nonempty ι] (a : M) : 𝔼 _i : ι, a = a := Finset.expect_const univ_nonempty _
+@[simp] lemma expect_const [Nonempty ι] (a : M) : 𝔼 _i : ι, a = a :=
```

Comment:
This is the same one that you're marking not simp in another PR, yes?

Reply:
Now that `univ_nonempty` is simp, I can indeed unsimp this PR